metadata.rb

```diff
@@ -21,6 +21,7 @@
 depends          'apt' # We recommend '>= 2.1.0'. See CHANGELOG.md for details
 depends          'chef_handler', '~> 1.1.0'
 depends          'yum'
+depends          'windows'
```

Comment:
The introduction of another cookbook dependency is something that is never "awesome" - are you certain the default providers in Chef 10, 11 and 12 cannot be used?